examples/with-react-intl/next.config.js

```diff
@@ -1,10 +0,0 @@
-const withTM = require('next-transpile-modules')([
-  '@formatjs/intl-relativetimeformat',
-  '@formatjs/intl-utils',
-  'react-intl',
-  'intl-format-cache',
-  'intl-messageformat-parser',
-  'intl-messageformat',
```

Comment:
None of these packages is shipping es6 imports anymore?

Reply:
Couple of issues:
- I actually don't know why this was needed. We do ship es6 imports but when I remove it everything still works.
- `next-transpile-modules` actually *re-runs* babel, which is an issue bc some babel plugins (in this case, ours) verify & modify AST in a way that double running can crap out.

Reply:
Hi! I added `next-transpile-modules` in #11849, since we noticed our app using react-intl and Next.js shipped `const` and other es6 stuff that broke older browsers. To verify, I used this snippet: `npm run build && npx es-check --module es5 ./.next/static/**/*.js`. I have no idea if this is needed anymore tho

Reply:
Yeah I tested in ie11 and it works without it